Sources/Mapper.swift

```diff
@@ -110,7 +110,7 @@ public struct Mapper {
         }
 
         let rawValues = try array.map { try T.RawValue.fromMap($0) }
-        return rawValues.flatMap { T(rawValue: $0) ?? defaultValue }
+        return rawValues.compactMap { T(rawValue: $0) ?? defaultValue }
```

Comment:
It won't build now for anyone that want to use this library with Swift 4.0.X, right?

Reply:
Yep, I'll wait for the final Xcode 9.3 release and once I merge this I will bump the major version.

Reply:
Yeah, I guess that's reasonable 👍